.ci/aarch64_linux/aarch64_wheel_ci_build.py

```diff
@@ -54,6 +54,14 @@ def build_ArmComputeLibrary() -> None:
     for d in ["arm_compute", "include", "utils", "support", "src"]:
         shutil.copytree(f"{acl_checkout_dir}/{d}", f"{acl_install_dir}/{d}")
 
+def replace_tag(filename):
```

Comment:
Do some type annotations please. And may be add a docstring
```suggestion
def replace_tag(filename: str) -> None:
```